src/Functions/createSimpleMPRWorldCoordinates.m

```diff
@@ -1,13 +1,22 @@
-%% Convenience Wrapper for the MPRWorldCoordinate Class by Florian Bernard
-%
-% Andreas Husch
-% Centre Hospitalier de Luxembourg, Dept. of Neurosurgery /
-% University of Luxembourg - Luxembourg Centre for Systems Biomedicne
-% 2017
-% mail@andreashusch.de, husch.andreas@chl.lu
 function [mpr, oc] = createSimpleMPRWorldCoordinates(nii)
-%mprPanel = uipanel('Parent', gcf);
-axesObj = gca;%axesObj = axes('Parent', mprPanel);
+% Convenience Wrapper for the MPRWorldCoordinate Class by Florian Bernard
+%
+% USAGE:
+%
+%    [mpr, oc] = createSimpleMPRWorldCoordinates(nii)
+%
+% INPUT:
+%    nii:       
```

Comment:
@adhusch, you need to complete these headers 😄

Reply:
nii: NiftiModality object to visualize as multi-planar-reformatting (MPR)
mar: Handle to MPR 
oc: optional orientation cube (deprecated)